test/features/email/presentation/extensions/email_extension_test.dart

```diff
@@ -0,0 +1,163 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email_header.dart';
+import 'package:jmap_dart_client/jmap/mail/email/individual_header_identifier.dart';
+import 'package:model/email/email_property.dart';
+import 'package:tmail_ui_user/features/email/presentation/extensions/email_extension.dart';
+import 'package:tmail_ui_user/features/email/presentation/model/smime_signature_status.dart';
+
+void main() {
+  group('EmailExtension::sMimeStatus::test', () {
+    test(
```

Comment:
please add the cases: ` Good signature`, ` Bad signature`, ` Good Signatures`, .....

Reply:
done